hack/create-upgrade-configmap.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+indent() {
+  INDENT="      "
+  sed "s/^/$INDENT/" | sed "s/^${INDENT}\($1\)/${INDENT:0:-2}- \1/"
+}
+
+NAME=${NAME:-"metering-ocp"}
+NAMESPACE=${NAMESPACE:-"openshift-metering"}
+
+ROOT_DIR=$(dirname "${BASH_SOURCE}")/..
+MANIFESTS_DIR="${MANIFESTS_DIR:-${ROOT_DIR}/manifests/deploy/openshift/olm/bundle/}"
+MANIFEST_VERSION=${MANIFEST_VERSION:-$(basename $(find $MANIFESTS_DIR -type d | sort -r | head -n 1))}
+
+CRD=$(sed '/^#!.*$/d' $MANIFESTS_DIR/$MANIFEST_VERSION/*crd.yaml | grep -v -- "---" | indent apiVersion)
+CSV=$(sed '/^#!.*$/d' $MANIFESTS_DIR/$MANIFEST_VERSION/*version.yaml | sed -e "s,imagePullPolicy: IfNotPresent,imagePullPolicy: Always,"  | sed 's/namespace: placeholder/namespace: '$namespace'/' |grep -v -- "---" |  indent apiVersion)
+PKG=$(sed '/^#!.*$/d' $MANIFESTS_DIR/*package.yaml | indent packageName)
+
+if [ -n "${IMAGE_OVERRIDE:-}" ] ; then
+    echo "Overriding the metering-operator deployment images with ${IMAGE_OVERRIDE}"
+    CSV=$(echo "$CSV" | sed -e "s~containerImage:.*~containerImage: ${IMAGE_OVERRIDE}~" | indent apiVersion)
+    CSV=$(echo "$CSV" | sed -e "s~image:.*~image: ${IMAGE_OVERRIDE}\n~" | indent ApiVersion)
+fi
+
+cat <<EOF | sed 's/^  *$//' | oc apply -n $NAMESPACE -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: $NAME
```

Comment:
To match up with the rest of the script and improves shell convention, these should be in the form `${VAR}`, right?